src/pages/react/component/index.md

```diff
@@ -52,6 +52,9 @@ class Cat extends React.Component {
 }
 
 ```
+Classful components are stateful components because they have a state associated with them, so they should be used wisely , 
```

Comment:
Hi @geekyatri 

Not all `Class Components` are inherently stateful. It depends on whether `state` is being defined in the component. Can you please rephrase this paragraph to explain this in detail?